aws/resource_aws_codedeploy_deployment_group_test.go

```diff
@@ -42,6 +42,9 @@ func TestAccAWSCodeDeployDeploymentGroup_basic(t *testing.T) {
 						"aws_codedeploy_deployment_group.foo", "service_role_arn",
 						regexp.MustCompile("arn:aws:iam::[0-9]{12}:role/foo_role_.*")),
 
+					resource.TestCheckResourceAttr(
+						"aws_codedeploy_deployment_group.foo", "blue_green_deployment_config.#", "0"),
```

Comment:
Since you're in here, you can clean up those `foo` bits to be our standard `test` naming.